test/src/helpers.ts

```diff
@@ -852,6 +860,7 @@ export function fundErc20(
   return async function fundErc20() {
     if (!erc20) erc20 = await freshERC20();
     for (const address of [options.account.address, ...(funded ?? [])]) {
+      console.log(`minting ${amount} to ${address}`);
```

Comment:
```suggestion
```

Reply:
I want to keep this in for now, at least during early dogfooding